addOns/help/src/main/javahelp/contents/releases/2.17.0.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+  Release 2.17.0
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Release 2.17.0</H1>
+
+This is a bug fix and enhancement release.
+
+TBC
+
+<H3>Structured Reports ISO 8601 Standard Date</H3>
+The structured reports (JSON and XML) generatedby ZAP via the Report Generation add-on now have an ISO 8601 standard date field/attribute (“created”); 
+the existing “generatedString” field may be removed in the future. (Reference: https://github.com/zaproxy/zap-extensions/pull/6711)
```

Comment:
- will be removed
- Not sure the reference adds anything more, if you want to keep it make it a link rather than plain text.